spec/system/support_interface/viewing_provider_permissions_spec.rb

```diff
@@ -37,6 +42,23 @@ def and_there_are_two_providers_in_a_partnership
     )
   end
 
+  def and_there_are_two_other_providers_in_a_partnership_with_no_permissions_configured
+    training = create(:provider, sync_courses: true, name: 'City Learning Trust')
+    ratifying = create(:provider, sync_courses: true, name: 'Staffordshire University (S72)')
+
+    create(
```

Comment:
This is a valid relationship (as far as permissions validation is concerned) and the only way any of these permissions would be true is if the training provider has completed the organisational permissions flow, except this would also have populated `setup_at`. Please see my other comment.